test/chi/custom-elements/buttons.pug

```diff
@@ -0,0 +1,197 @@
+---
+title: Buttons
+scripts:
+  - js/ce/ux-chi-ce
+---
+each size in ['default', 'sm', 'md', 'lg', 'xl']
+  each type in ['solid', 'outline', 'flat', 'fluid']
+    div(class={
+      [`test-${type}`]: size === 'default',
+      [`test-${type}-${size}`]: size !== 'default'
+    })
+      span.a-h2= `Button / ${type} / ${size}`
+      table(cellpadding="0", cellspacing="0")
+        each state in ['initial', 'hover', 'active', 'focus', 'disabled']
+          tr
+            td
+              small= `${state}`
+            each style in ['', 'primary', 'secondary', 'danger', 'dark', 'light']
```

Comment:
@srinath-koneru Please, include `inverse` type.

Reply:
@srinath-koneru , please run tests: 
```
npm run test
```
Review reports in: 
```
reports/html_report/non_responsive/index.html
reports/html_report/non_responsive_ce/index.html
reports/html_report/responsive/index.html
```
As it is not being run in docker, yours might differ from the reference ones so approve them: 
```
npm run approve
```
Then, run tests again. You can now run CE test only to save time: 
```
npm run test-ce
```
Review CE tests in:
```
reports/html_report/non_responsive_ce/index.html
```
This tests should look exactly the same as the reference ones.